### DISCUSSION TITLE
有没有现成的工具可以实现模仿真实上网，隐藏一切远程信息？

### DISCUSSION THREAD

**Discussion by lucas-yang256** [v2fly/v2ray-core#2873] [Ideas]
我的需求是，想要弄一个翻墙工具，不仅有代理的功能，还能够100%模仿真实的上网行为。比如说，现在的V2RAY只是简单的信息转发，信息报文当中还包含了client请求的信息。这就会引发一个问题：被请求的网络服务器（比如google），就能够根据报文信息和报文发送方的信息一致性来判断是否使用了代理，以及原始的请求方所在的时区等等。这样的话，虽然有时不担心位置信息泄露问题，但是用google bard就会被探测出来所在的国家，被禁用了。而如果能够在信息转发之前先将请求信息进行修改（修改的信息可以有：发送时间，发送方电脑型号，等等），那么就无需担心这个问题了。

目前，V2RAY有这个功能吗？或者说有人知道有没有现成的工具可以实现此功能吗？

**Comment by puzzle9** [comment]
你这把问题抬到了不属于他的高度

我觉得 你或许需要一个 vps 远程连过去
用它上网
